src/Symfony/Component/Translation/Extractor/PhpExtractor.php

```diff
@@ -236,6 +242,10 @@ protected function parseTokens($tokens, MessageCatalogue $catalog)
 
                 if ($message) {
                     $catalog->set($message, $this->prefix.$message, $domain);
+                    $metadata = $catalog->getMetadata($message, $domain) ?? [];
+                    $normalizedFilename = preg_replace('{[\\\\/]+}', '/', $filename);
```

Comment:
Can't we use `str_replace('\\', '/', $filename)` instead?

Reply:
I also replace consecutive directory separators in order to solve the issue on line 4021: https://travis-ci.org/symfony/symfony/jobs/524492379 without changing `resourcesProvider()`. Alternatively we can use `str_replace('\\', '/', $filename)` and reintroduce the changes to `resourcesProvider()` (remove trailing `/` of `$directory` and add it to the `glob()` calls).